src/api/index.ts

```diff
@@ -51,23 +51,24 @@ export class Api {
   static get host(): string {
     return import.meta.env.PROD
       ? (import.meta.env.VITE_API_URL as string)
-      : 'http://localhost:3000';
+      : 'http://api.resurrectism.test:3000';
   }
 
   static async fetch<Body extends Record<string, unknown>>(
     url: string,
     method: string,
     data?: Body,
   ) {
-    const options = {
+    const res = await fetch(`${Api.host}${url}`, {
       method,
+      mode: 'cors',
       headers: {
         'Content-Type': 'application/vnd.api+json',
         Authorization: `Bearer ${localStorage.getItem('token')}`,
```

Comment:
Do we actually need to get the `token` from local storage ? Don't we use tokens with http cookies ?